src/item.cpp

```diff
@@ -1260,6 +1262,98 @@ static void insert_separation_line( std::vector<iteminfo> &info )
     }
 }
 
+double item::effective_dps( const player &guy, monster &mon ) const
+{
+    const float mon_dodge = mon.get_dodge() * 5.0f;
+    float base_hit = guy.get_hit_base() + guy.get_hit_weapon( *this );
+    base_hit *= std::max( 0.25f, 1.0f - guy.encumb( bp_torso ) / 100.0f );
+    float mon_defense = mon_dodge + mon.size_melee_penalty();
+    double num_hits = 0;
+    constexpr double hit_trials = 1000.0;
+    const float accuracy = base_hit * 0.5f;
+    // monte carlo determination of the number of hits because calculating odds against normal
+    // rolls is hard
+    for( int i = 0; i < hit_trials; i++ ) {
+        num_hits += ( normal_roll( accuracy, 25.0f ) > mon_defense ? 1.0 : 0.0 );
+    }
```

Comment:
I don't think Monte Carlo should be necessary here.  I think it's something like:
```
double num_hits = std::cerf( (mon_defense - accuracy ) / 25 / M_SQRT_2 ) / 2 * hit_trials;
```
(although that should probably be wrapped up in a `right_tail_normal` function).

Reply:
Replaced with a lookup table for consistency.